app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -95,7 +95,18 @@ class MainActivity : AppCompatActivity() {
     private var extensionModeIndex = 0
     private var supportedExtensionModes = listOf(extensionMode)
 
+    // Always add the secure intents first
+    // They must be checked before the non-secure ones as they won't
+    // trigger the request to dismiss the keyguard
+    // Must be sorted by priority
+    private val secureIntentsMap = mapOf(
+        Pair(MediaStore.ACTION_REVIEW_SECURE, true),
```

Comment:
Now that I'm looking at this, we can do
ACTION to $bool
To use the kotlin extension to create pairs